docs/schemas/core/IfcProductExtension/Entities/IfcBridgePart.md

```diff
@@ -10,17 +10,15 @@ Part of a bridge.
 
 ### Product Local Placement
 
-The local placement for IfcBridgePart is defined in its supertype IfcProduct. It is defined by the IfcLocalPlacement or IfcLinearPlacement, which defines the local coordinate system that is referenced by all geometric representations.
+The local placement for _IfcBridgePart_ is defined in its supertype _IfcProduct_. It is defined by the _IfcLocalPlacement_ or _IfcLinearPlacement_, which defines the local coordinate system that is referenced by all geometric representations.
 
-* The PlacementRelTo relationship of IfcLocalPlacement or IfcLinearPlacement shall point (if relative placement is used) to the IfcSpatialStructureElement of type IfcBridge, or of type IfcBridgePart (e.g. to position a bridge part relative to a bridge part complex, or a partial bridge part to a bridge part).
+* The PlacementRelTo relationship of _IfcLocalPlacement_ or _IfcLinearPlacement_ shall point (if relative placement is used) to the _IfcSpatialStructureElement_ of type _IfcBridge_, or of type _IfcBridgePart_ (e.g. to position a bridge part relative to a bridge part complex, or a partial bridge part to a bridge part).
```

Comment:
Similar comment as with the IfcBridge documentation regarding the IfcLinearPlacement.